pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/RealtimeProvisioningHelperCommand.java

```diff
@@ -158,17 +173,36 @@ public boolean execute()
       throw new RuntimeException("Exception in reading table config from file " + _tableConfigFile, e);
     }
 
+    StringBuilder note = new StringBuilder();
+    note.append("\nNote:\n");
     int numReplicas = tableConfig.getValidationConfig().getReplicasPerPartitionNumber();
-    if (_retentionHours == 0) {
-      if (tableConfig.getValidationConfig().getSegmentPushFrequency().equalsIgnoreCase("hourly")) {
-        _retentionHours = DEFAULT_RETENTION_FOR_HOURLY_PUSH;
+    int tableRetentionHours = (int) TimeUnit.valueOf(tableConfig.getValidationConfig().getRetentionTimeUnit())
+            .toHours(Long.parseLong(tableConfig.getValidationConfig().getRetentionTimeValue()));
+    if (_retentionHours > 0) {
+      note.append("\n* Table retention and push frequency ignored for determining retentionHours");
```

Comment:
can we add "since retentionHours is provided" to this message. Otherwise it looks like it is flagging something that went wrong

Reply:
ok